packages/pigeon/lib/pigeon_lib.dart

```diff
@@ -282,6 +304,16 @@ class Pigeon {
       ));
     }
 
+    for (final ClassMirror enumMirror in enums) {
+      List<String> members = <String>[];
+      // The first 3 declarations and last two are trimmed as they are innate to
+      // enums in Dart and are not user defined values.
+      for (int i = 3; i < enumMirror.declarations.keys.length - 2; i++) {
```

Comment:
fwiw this could be more compactly written as `members = enumMirror.declarations.keys.drop(2).map(MirrorSystem.getName);`  (assuming there is a drop function, I think there is.)